lunary/__init__.py

```diff
@@ -1455,6 +1454,49 @@ async def get_raw_template_async(slug: str, app_id: str | None = None, api_url:
     templateCache[slug] = {'timestamp': now, 'data': data}
     return data
 
+def get_templates(live_only: Optional[bool] = False, app_id: Optional[str] = None, api_url: Optional[str] = None):
```

Comment:
this is linked to another feature right? I don't asking you to do that, was there as reason to implement it?